modules/repo_sync/src/Controller/RepoSyncController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Drupal\devportal_repo_sync\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Url;
+use Drupal\devportal_repo_sync\Service\Client;
+
+/**
+ * Class RepoSyncController.
+ */
+class RepoSyncController extends ControllerBase {
+
+  /**
+   * Creates a table from the imported repositories.
+   *
+   * @return array
+   *   Return table array.
+   */
+  public function content() {
+    $config = $this->config('devportal_repo_sync.config');
+    $client = new Client($config->get('uuid'), hex2bin($config->get('secret')), "http://service:8000");
+
+    try {
+      $result = $client("GET", "/api/import", NULL);
+      $result = json_decode(array_pop($result));
+
+      foreach ($result->items as $item) {
+        $rows[] = [
```

Comment:
The `$rows` variable should be defined before the foreach, like this: `$rows = [];`